### ISSUE TITLE
Help Extensions not working in Symfony 2.3.3

### ISSUE THREAD

**Issue by ColinMacD** [simplethings/SimpleThingsFormExtraBundle#44]
I think field is no longer and extendible form type and form should be used instead. If I make the change myself I no longer get errors, but the help messages doesn't render in my template.

**Comment by bobvandevijver** [comment]
In Resources/views/Form/field_type_help.html.twig change

```
{% block field_row %}
  {# rest of code #}
{% endblock field_row %}
```

to

```
{% block form_row %}
  {# rest of code #}
{% endblock form_row %}
```

**Comment by bobvandevijver** [comment]
@ColinMacD Or write your own "help" extension, it is not so difficult, and more flexibel.

**Comment by ColinMacD** [comment]
Thanks for the response, I'll give that a go.

**Comment by DavidBadura** [comment]
@ColinMacD fixes the pull request #45 your problem?

**Comment by smoench** [comment]
has been fixed
